cibuildwheel/windows.py

```diff
@@ -174,9 +176,24 @@ def setup_python(
         )
         sys.exit(1)
 
-    # make sure pip is installed
+    log.step("Installing build tools...")
+
+    # make sure pip is installed and available on PATH
     if not (installation_path / "Scripts" / "pip.exe").exists():
-        call(["python", get_pip_script, *dependency_constraint_flags], env=env, cwd="C:\\cibw")
+        # perhaps pip is installed, but not available as 'pip.exe'...
+        pip_is_installed = (
+            call(
+                ["python", "-m", "pip", "--version"], env=env, cwd="C:\\cibw", check=False
+            ).returncode
+            == 0
+        )
```

Comment:
This bit is kinda clumsy, as is adding the return type on `call()`... maybe the following is better?

```suggestion
        try:
            call(["python", "-m", "pip", "--version"], env=env, cwd="C:\\cibw")
        except subprocess.CalledProcessError:
            pip_is_installed = False
        else:
            pip_is_installed = True
```

Reply:
Yeah I'm gonna change it…